### PR TITLE
chore(gatsby-remark-autolink-headers): add styling instructions

### LINKED ISSUE #20982: Add how to style remark-autolink-headers anchor icon
## Summary
It's unclear how to style the SVG that is generated by https://www.gatsbyjs.org/packages/gatsby-remark-autolink-headers/. This was raised on https://github.com/gatsbyjs/gatsby/pull/20968.

We should add a small section to the readme that explains how to do this with css.

### Motivation

Users who use the gatsby-remark-autolink-headers plugin.

## Steps to resolve this issue

Edit [Readme.md of gatsby-remark-autolink-headers](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-remark-autolink-headers/README.md) and add a section how to style the anchor svg

### Draft the doc

- [ ] Write the doc, following the format listed in these resources:
  - [Overview on contributing to documentation](https://www.gatsbyjs.org/contributing/docs-contributions/)

### Open a pull request

- [ ] Open a pull request with your work including the words "closes #[this issue's number]" in the pull request description

### PR DESCRIPTION
**Pull request by Lundbyit**
## Description

Add instructions on how to style svg in gatsby-remark-autolink-headers

## Related Issues

Fixes #20982

### CODE REVIEW

**Comment by wardpeet** [comment]
@Lundbyit can you be a little more descriptive? The comment I wrote was a solution but it needs some work to make it resonate with people. Draft problem statement, ...

**Comment by Lundbyit** [comment]
@wardpeet, alright. I'll make it more descriptive.

**Comment by Lundbyit** [comment]
Am I descriptive enough or should I add more text to the documentation? @wardpeet

File: packages/gatsby-remark-autolink-headers/README.md [hunk 1]
```diff
@@ -87,3 +87,23 @@ module.exports = {
...
+}
+```
+
+If you want to only target the svg (note that the first selector is faster than the one below but that's micro optimisations)
+
+```
+a.anchor > svg {
+    fill: 'red'
+}
+```
```
Comment (wardpeet) [inline packages/gatsby-remark-autolink-headers/README.md:109]:
````suggestion
By default, the anchor link has no additional styling. To make it fit your website, we'll have to write some extra CSS rules. 

If you want to make the anchor icon red, you can add the following CSS rule:

```
a.anchor {
  fill: 'red'
}
```

note: Not sure how to add custom css? You can visit our [tutorial][https://www.gatsbyjs.com/tutorial/part-two/).
````

**Comment by Lundbyit** [comment]
Hi, I'm sorry you had to write the docs.

**Review by LekoArts — APPROVED** [review]
👍

**Pull request merged**